cheddar/cheddar-integration-aws/src/main/java/com/clicktravel/infrastructure/persistence/aws/dynamodb/DynamoDocumentStoreTemplate.java

```diff
@@ -311,7 +316,7 @@ private PrimaryKey getPrimaryKey(final ItemId itemId, final ItemConfiguration it
         return value.toString();
     }
 
-    private <T extends Item> T stringToItem(final String item, final Class<T> valueType) {
+    <T extends Item> T stringToItem(final String item, final Class<T> valueType) {
```

Comment:
yes, it allows the method to be used in the unit test for converting between items and JSON strings to set up the expectations of mock return values.